patches/lua-tree.patch

```diff
@@ -42,6 +46,8 @@ schema/init.lua:
 - use re_match as UUID match intad of re_find
 - remove log statements
 - ensure custom_entity_checks in schemas return error message
+- Reintroducing fallback Lua functions, as the OpenResty tooling is not available for use.
+  Read more: https://github.com/Kong/kong/commit/15c427e090452396a5f52bf6848d8ad6bbe5e2d1
```

Comment:
Secret referencing has been disabled.

Reply:
Sorry, can you be a bit more specific? Are you saying to add that as a bullet item? I felt that was not needed as this comment is saying the same thing:
https://github.com/Kong/goks/blob/c3b4a7a59bcddd38f12954c63ad2ae47d1d90ea9/patches/lua-tree.patch#L40